static/rison.js

```diff
@@ -138,7 +144,12 @@
             },
             s = {
                 array: function (x) {
-                    var a = ['!('], b, f, i, l = x.length, v;
+                    var a = ['!('],
+                        b,
+                        f,
+                        i,
+                        l = x.length,
+                        v;
```

Comment:
Same here

Reply:
argh yeah good spot. I'll ignore this one (or come up with something here)